inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/graph/RdfUtils.java

```diff
@@ -43,10 +43,10 @@
 public class RdfUtils
 {
     public static Optional<Statement> readFirst(RepositoryConnection conn, Resource subj, IRI pred,
-            Value obj, String language, KnowledgeBase aKB)
+            Value obj, KnowledgeBase aKB)
     {
-        try (RepositoryResult<Statement> results =
-            getStatementsSparql(conn, subj, pred, obj, false, language, aKB)) {
+        try (RepositoryResult<Statement> results = getStatementsSparql(conn, subj, pred, obj, false,
```

Comment:
Is the null dangerous here?

Reply:
No - `SPARQLQueryStore.languageFilter(String, String)` (where this eventually arrives) allows the language to be null. Anyway, this code is hopefully going away soon in favor for `SPARQLQueryBuilder`.